### ISSUE TITLE
[a11y]: Users view - accessibility findings

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3895]
### Why is this issue important?

This ticket contains multiple accessibility issues in the Uploads view and subview in arpa_reporter.  As support for Arpa reporter is winding down we collected issues in one page.  

Note:  These were identified during a developer accessibility review to double check the accessibility of the elements on the page.   To fully insure the site is accessible to users, more comprehensive testing by an accessibility subject matter expert should be performed.

Testing approach
All testing was done against [staging](https://staging.grants.usdr.dev/arpa_reporter) and local dev.
1) Run [axe DevTools chrome extension ](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)against the page.   This checks contrast 
2) Run through the page with the keyboard - I used Microsoft's [Accessibility Insights Chrome Extension](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to track keyboard focus
3) Zoom the page to 200x and 400x to check reflow
4) Run through the page using a screen reader - mac voice over
5) Validate the HTML by pasting parts of the html into [W3C Markup Validation Service](https://validator.w3.org/#validate_by_input)

Testing gaps
1) Windows screen readers - JAWS / NVDA
2) Mobile web devices - the arpa reporter was not designed to be responsive
3) Real user workflow steps.   

----
# Users view

## Issue 1 - edit button should have aria-label that provides an identifier of what will be edited.  e.g. "Edit Mindy Huang"

![Image](https://github.com/user-attachments/assets/67f930ab-5009-42ae-93ee-dcb837e890fe)

## Issue 2 - Focus indicator is missing when tabbing to column headings for sort

![Image](https://github.com/user-attachments/assets/57a85954-df02-48f2-bfe0-48dca123b5c6)

### Current State

See individual issues above

### Expected State

See individual issues above

### Implementation Plan

TBD

### Relevant Code Snippets

```shell

```
